indico/modules/events/registration/fields/base.py

```diff
@@ -128,7 +128,7 @@ def process_field_data(cls, data, old_data=None, old_versioned_data=None):
         """
         data = dict(data)
         if 'places_limit' in data:
-            data['places_limit'] = int(data['places_limit']) if data['places_limit'] else 0
+            data['places_limit'] = data.get('places_limit', 0)
```

Comment:
I guess this is guaranteed to be an int now since it's coming from a marshmallow schema?
Also, no need for `.get()` here since it's inside an `if 'places_limit' in data:` block..

Reply:
Yeah. Actually, isn't this whole `if` block pointless?

Reply:
Depends... if it's used in a PATCH-type update where only modified data is present, then this would be needed to not overwrite data. But I think for field setup we always send the whole settings, so I think it's indeed pointless.